pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1670,6 +1670,54 @@ func (n *NetworkPolicyController) cleanupOrphanGroups(internalNetworkPolicy *ant
 	}
 }
 
+// verifyPoliciesProcessed checks that all the policy objects in the cluster supported by Antrea
+// are already processed by the Antrea controller. It serves as a sanity check/prerequisite for
+// the networkpolicyanalysis command, since the command uses controller cache as computation
+// source. Note that:
+//  1. This function blocks internal NP processing until it returns, which is acceptable given
+//     the rarity of invocation.
+//  2. Verification does not guarantee the latest version of the policy is processed.
+//  3. Verification is based on the fact that, as of now, processed internal NPs and original
+//     policy has a one-to-one relationship.
+func (n *NetworkPolicyController) verifyPoliciesProcessed() (bool, error) {
+	n.internalNetworkPolicyMutex.Lock()
+	defer n.internalNetworkPolicyMutex.Unlock()
+
+	numInternalNP := n.GetNetworkPolicyNum()
+	policyObjNum := 0
+	if npList, err := n.networkPolicyLister.List(labels.Everything()); err != nil {
+		return false, err
+	} else {
+		policyObjNum += len(npList)
+	}
+	if acnpList, err := n.acnpLister.List(labels.Everything()); err != nil {
+		return false, err
+	} else {
+		policyObjNum += len(acnpList)
+	}
+	if annpList, err := n.annpLister.List(labels.Everything()); err != nil {
+		return false, err
+	} else {
+		policyObjNum += len(annpList)
+	}
+	if features.DefaultFeatureGate.Enabled(features.AdminNetworkPolicy) {
```

Comment:
Inspired by this line, `features.AntreaPolicy` could be disabled, so do we need to check if that is enabled previously?